src/WorkerMessageTypes.d.ts

```diff
@@ -173,16 +172,16 @@ export interface GroupCreateWorkerRequest {
     type: "GROUP_CREATE";
     message: {
         userPublicKey: PublicKey<Uint8Array>;
-        addAsMember: boolean;
         signingKeys: SigningKeyPair;
+        memberList?: UserOrGroupPublicKey[];
     };
 }
 export interface GroupCreateWorkerResponse {
     type: "GROUP_CREATE_RESPONSE";
     message: {
         encryptedGroupKey: PREEncryptedMessage;
         groupPublicKey: PublicKey<Uint8Array>;
-        transformKey?: TransformKey;
+        transformKeyGrantList?: TransformKeyGrant[];
```

Comment:
Lets change all of the places that take `transformKeyGrantList` as optional to just be required. Because it can be provided as an empty array and still be valid for the type, lets make it concrete. This change should spider out into various places including the request layer.